src/Cli/Sake.php

```diff
@@ -107,6 +107,8 @@ public function run(?InputInterface $input = null, ?OutputInterface $output = nu
                 }
                 $this->kernel->boot($flush);
             }
+            // Register sake in the injector so we can access it from anywhere in the execution chain
+            Injector::inst()->registerService($this);
```

Comment:
```suggestion
            Injector::inst()->registerService($this, Sake::class);
```
For the unlikely though possible scenario where someone subclasses Sake

Reply:
Done